app/uk/gov/hmrc/tariffclassificationfrontend/connector/RulingConnector.scala

```diff
@@ -17,20 +17,19 @@
 package uk.gov.hmrc.tariffclassificationfrontend.connector
 
 import javax.inject.{Inject, Singleton}
-import uk.gov.hmrc.http.{HeaderCarrier, HttpReads}
+import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.bootstrap.http.HttpClient
 import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 @Singleton
-class RulingConnector @Inject()(configuration: AppConfig,
-                                http: HttpClient) {
+class RulingConnector @Inject()(configuration: AppConfig, http: HttpClient) extends WithAuth {
 
-  def notify(id: String)(implicit hc: HeaderCarrier): Future[Unit] = {
-    val authenticatedHeaders: HeaderCarrier = hc.withExtraHeaders("X-Api-Token" -> configuration.apiToken)
-    http.POSTEmpty(s"${configuration.rulingUrl}/binding-tariff-rulings/ruling/$id")(HttpReads.readRaw, authenticatedHeaders, global).map(_ => ())
+  def notify(id: String, hc: HeaderCarrier): Future[Unit] = {
```

Comment:
I prefer this as implicit, then we dont have visible header carrier code in the services